public/app/plugins/datasource/prometheus/configuration/AzureCredentialsForm.tsx

```diff
@@ -148,6 +149,11 @@ export const AzureCredentialsForm = (props: Props) => {
       onCredentialsChange(updated);
     }
   };
+  const prometheusConfigOverhaulAuth = config.featureToggles.prometheusConfigOverhaulAuth;
+
+  const inputWidth2030 = prometheusConfigOverhaulAuth ? 'width-20' : 'width-30';
```

Comment:
Here also, things like `cx` could help do those conditional classes thing. But I guess this PR is not about fixing something that was already broken

non blocking

Reply:
Added!